config/local/generate_toml.go

```diff
@@ -64,125 +65,19 @@ var LocalConfig = config.OracleConfig{
 		// all of the currency pairs. Before adding a new market to the oracle, ensure that
 		// the provider supports fetching data for the currency pair.
 		{
-			// -----------------------------------------------------------	//
-			// ---------------------Start Binance API----------------------	//
-			Name: binance.Name,
-			API:  binance.DefaultUSAPIConfig,
-			Market: config.MarketConfig{
-				Name: binance.Name,
-				CurrencyPairToMarketConfigs: map[string]config.CurrencyPairMarketConfig{
-					"BITCOIN/USD": {
-						Ticker:       "BTCUSDT",
-						CurrencyPair: oracletypes.NewCurrencyPair("BITCOIN", "USD"),
-					},
-					"ETHEREUM/USD": {
-						Ticker:       "ETHUSDT",
-						CurrencyPair: oracletypes.NewCurrencyPair("ETHEREUM", "USD"),
-					},
-					"ATOM/USD": {
-						Ticker:       "ATOMUSDT",
-						CurrencyPair: oracletypes.NewCurrencyPair("ATOM", "USD"),
-					},
-					"SOLANA/USD": {
-						Ticker:       "SOLUSDT",
-						CurrencyPair: oracletypes.NewCurrencyPair("SOLANA", "USD"),
-					},
-					"ETHEREUM/BITCOIN": {
-						Ticker:       "ETHBTC",
-						CurrencyPair: oracletypes.NewCurrencyPair("ETHEREUM", "BITCOIN"),
-					},
-				},
-			},
+			Name:   binance.Name,
```

Comment:
Moving the market configs into each provider dir, this config is getting to big kek.